scripts/jenkins/set-node-version.sh

```diff
@@ -3,5 +3,7 @@
 # Install nvm for this shell
 source ~/.nvm/nvm.sh
 
-# Use node.js at 4.2.x
-nvm install 4.4
+# Use version of node.js found in .nvmrc
+nvm install
+
+npm install -g yarn
```

Comment:
Nit: This file now does more than its name suggests. Maybe rename it to something like `setup-node-env`?